scripts/UpdateCopyrightHeader.py

```diff
@@ -178,16 +219,55 @@ def run(args):
     if line.startswith(" * @copyright"):
         if "DigitalGlobe" not in line:
             copyrights.append(line.rstrip())
+        elif "DigitalGlobe" in line: # New condition. Purpose: extract existing copyright years. (By Dave Emmith, 2-2-2017)
+            # Set the pattern to get the years from ...  * @copyright Copyright (C) 2012, 2013 DigitalGlobe (http://www.digitalglobe.com)
+            p = re.compile("\(C\)\s*(.*?)\s*DigitalGlobe")
+            # Search 'line' for the pattern.
+            m = p.search(line)
+            # If there is a match then assign it to existingCopyrightYears
+            if m:
+                existingCopyrightYears = m.group(1)
+                #print "existingCopyrightYears: " + existingCopyrightYears
     lineCount += 1
     oldHeaderBlock += line
   else:
     #print "Augmenting line!"
     lowerNewFileStr += line
     lineCount += 1
-
+    
+#if existingCopyrightYears != "":
+#    print "existingCopyrightYears: " + existingCopyrightYears
+#else:
+#    print "existingCopyrightYears: NONE found\t" + options.fileName
```

Comment:
Thanks Ben, I'll be reviewing that as there is a minor change that needs to be made too.